conf/modules.config

```diff
@@ -202,6 +202,23 @@ process {
             enabled: false,
         ]
     }
+    withName: ENSEMBLVEP {
```

Comment:
Maybe we should name it differently as we will use vep for SNV/INDELS and Mitochondria as well. Like ENSEMBLVEP_SV?

Reply:
Yeah, good point! Will fix it in the next commit.